packages/dds/tree/src/core/tree/chunk.ts

```diff
@@ -15,12 +15,31 @@ import {
 } from "./cursor.js";
 import { rootFieldKey } from "./types.js";
 
+/**
+ * Properties for incremental summarization of the chunks. This is used to determine if this chunk has changed since
+ * the last summary.
+ * If it has changed, the chunk will be summarized.
+ * If it hasn't changed, the state will be used to incrementally summarize it by referring to the summary for this
+ * chunk in the previous summary.
+ */
+export interface IncrementalSummaryProps {
+	/**
+	 * The reference ID of the summary for this chunk in the previous summary.
+	 */
+	readonly summaryRefId: string | undefined;
```

Comment:
summaryRefId strings are used in several places,, created in few places and it would be bad if they were mixed up with other strings. This makes them an ideal candidate for a branded type which can improve type safety, and give a centralized place to document their semantics.

See FieldKey and many other usages of our `Brand` utility for examples.

Reply:
This is gone from the latest iteration as per our discussion.